rancher/v1.5/en/kubernetes/index.md

```diff
@@ -10,9 +10,20 @@ lang: en
+
+Kubernetes can be configured while creating or editing environment templates. If you start with a **Cattle** environment, you can configure and launch **Kubernetes** from the **Catalog** -> **Library**. If you choose the catalog route, you can skip step 1.
+
+1. Edit the Kubernetes configuration in an [environment template]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/#what-is-an-environment-template), click on **Edit Config** in the **Orchestration** section of the template. You can edit the configuration when creating a new environment templates or when editing an existing environment template.
+2. Confirm the template version of Kubernetes is the one that you'd like to use.
+3. Select the configuration options for the [plane isolation]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/kubernetes/resiliency-planes/), [cloud providers]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/kubernetes/providers/), [backups]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/kubernetes/backups/), [add-ons]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/kubernetes/addons/).
+4. Click on **Configure** to save the configuration options for the environment template of click on **Launch** to launch Kubernetes from the catalog.
+
+> **Note:** We recommend starting your Kubernetes environments with the correct configuration, if you want to change the configuration of an existing Kubernetes setup, you can click on **Up to Date** button to upgrade the Kubernetes setup to your new one. 
+
 ### Creating a Kubernetes Environment
 
-In the dropdown of environments, click on the **Manage Environments**. To create a new environment, click on **Add Environment**, provide a **Name**, **Description** (Optional), and select an environment template that has _Kubernetes_ as the orchestration. If [access control]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/configuration/access-control/) is turned on, you can [add members]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/#editing-members) and select their [membership role]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/#membership-roles). Anyone added to the membership list would have access to your environment.
+In the dropdown of environments, click on the **Manage Environments**. To create a new environment, click on **Add Environment**, provide a **Name**, **Description** (Optional), and select an environment template that has **Kubernetes** as the orchestration configured as you'd like. If [access control]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/configuration/access-control/) is turned on, you can [add members]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/#editing-members) and select their [membership role]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/#membership-roles). Anyone added to the membership list would have access to your environment.
```

Comment:
Should we merge the "configuring kubernetes" section into this? Or maybe make it more obvious that this workflow doesn't allow the user to configure kubernetes.